styles.css

```diff
@@ -0,0 +1,82 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+ul,
+a {
+  text-decoration: none;
+  list-style: none;
+}
+
+.container {
+  margin: auto;
+  width: 375px;
+  height: 718px;
+  border: 1px solid rebeccapurple;
+  border-bottom-left-radius: 7%;
+}
```

Comment:
- [ ] Kindly remove the border you have around your design as they aren't present in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/198430677-41cd76c5-3fed-466b-9a02-7b690a205b0a.png)  |  ![image](https://user-images.githubusercontent.com/74814780/198430575-39f14e19-d8a8-403c-b2d4-43aa3511a085.png)